src/Tvheadend.h

```diff
@@ -254,12 +246,14 @@ class ATTR_DLL_LOCAL CTvheadend : public kodi::addon::CInstancePVRClient,
   /*
    * VFS (pass-thru)
```

Comment:
"pass-thru" isn't really valid anymore, could be removed

Reply:
will do